schemas/rum/view-schema.json

```diff
@@ -362,6 +362,30 @@
                 "readOnly": true
               },
               "readOnly": true
+            },
+            "replay_stats": {
+              "type": "object",
+              "description": "Debug metadata for Replay Sessions",
+              "properties": {
+                "records_count": {
+                  "type": "integer",
+                  "description": "The amount of records produced during this view lifecycle",
```

Comment:
What is the view lifecycle? We also have a term `view lifetime` (used in vitals), we need probably to align naming and clarify what does it mean.